great_expectations/cli/batch_request.py

```diff
@@ -137,7 +137,7 @@ def select_data_connector_name(
             f"    {i}. {data_connector_name}"
             for i, data_connector_name in enumerate(data_connector_names, 1)
         ]
-    )
+    ) + "\n"
```

Comment:
```suggestion
    )
choices += "\n" # Necessary for consistent spacing between prompts
```
Since the `choices` object is already a bit complex with the usage of `join` and a list comprehension, could we move this newline addition to a separate line and just add a small comment?

Very much a nitpicky comment just trying to make it very clear what's changed :)

Reply:
Ah I see now, thanks for pointing out the style guide. I'll fix it shortly and test the styling as well.